mypy/nodes.py

```diff
@@ -587,6 +595,11 @@ class Var(SymbolNode, Statement):
     # parse for some reason (eg a silenced module)
     is_suppressed_import = False
 
+    FLAGS = [
+        'is_self', 'is_ready', 'is_initialized_in_class', 'is_staticmethod',
+        'is_classmethod', 'is_property', 'is_settable_property', 'is_suppressed_import'
```

Comment:
It still hasn't become second nature for you to add a trailing comma to a list like this, when the close bracket is on the next line? (Don't push a fix, just noticing. :-)